api/v1beta1/flinkcluster_types.go

```diff
@@ -387,6 +387,10 @@ type FlinkClusterSpec struct {
 	// Flink image spec for the cluster's components.
 	Image ImageSpec `json:"image"`
 
+	// BatchSchedulerName specifies the batch scheduler name for JobManager, TaskManager.
+	// If empty, no batch scheduling is enabled.
+	BatchSchedulerName string `json:"batchSchedulerName,omitempty"`
```

Comment:
Change the type to `*string` to make it optional explicitly.

Reply:
Do we want to distinguish "" and unset?

Reply:
I think both means no batch scheduling.

Reply:
So do i need to fix it?

Reply:
Yes, please. You can follow the example of `fromSavepoint` which is also an optional string field.

Reply:
thanks for the quick response, will do